cmd/jag/commands/run.go

```diff
@@ -5,20 +5,34 @@
 package commands
 
 import (
-	"errors"
+	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 
 	"github.com/blakesmith/ar"
 	"github.com/google/uuid"
 	"github.com/spf13/cobra"
 	"github.com/toitlang/jaguar/cmd/jag/directory"
 )
 
+// Checks whether a file is a snapshot file.  Actually only checks for an ar
+// file, since snapshot files are ar files.
```

Comment:
Snapshots have a 'toit' file with content 'like a tiger'.
You could check for that.
I think it would be ok to guarantee that this file is always in the beginning.

Reply:
OK I changed it to check for the toit archive member.